EzMap.Domain/Models/Poi.cs

```diff
@@ -16,4 +16,7 @@ public Poi(string name, string address, Guid userId)
     public string Name { get; set; }
     public Guid UserId { get; set; }
     public User User { get; set; }
+
+    public List<Collection> ListOfPois { get; } = new();
```

Comment:
This is a misleading name. This is a list of collections of Pois, not just 1 list.